CONTRIBUTING.md

```diff
@@ -148,6 +148,43 @@ for details.
 [cmake-prefix-path]: https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html
 [gtest]: https://github.com/google/googletest/
 
+### C/C++ with Meson
+
+While CMake is the officially supported build generator, there is limited,
+experimental support for the Meson build system. Meson offers arguably better
+dependency management than CMake, with a syntax that Python developers may
+find more readable.
+
+To use Meson, start at the c directory and run:
+
+```shell
```

Comment:
```suggestion
```console
```

Reply:
No problem making this change but all of the other entries in the CONTRIBUTING.md guide currently use `shell` - do you want me to update all of them as part of this or leave that to a separate PR?

Reply:
Oh. Could you use a separate PR?